library/core/src/time.rs

```diff
@@ -1129,10 +1129,10 @@ impl fmt::Debug for Duration {
 
                     // If the user request a precision > 9, we pad '0's at the end.
                     let w = f.precision().unwrap_or(pos);
```

Comment:
```suggestion
                    let width = f.precision().unwrap_or(pos);
```
for better legibility?